src/github.com/getlantern/lantern-android/libflashlight/bindings/go_bindings/go_bindings.go

```diff
@@ -10,21 +10,21 @@ import (
 )
```

Comment:
This file is generated by the build process, we should probably not take it into the source tree as it is not really editable.